src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -154,7 +155,9 @@ private ObjectNode populateBidRequestExtension(ObjectNode ext) {
     private JsonNode populatePriceGranularity(JsonNode priceGranularityNode, boolean isPriceGranularityNull,
                                               boolean isPriceGranularityTextual) {
         if (isPriceGranularityNull) {
-            return Json.mapper.valueToTree(PriceGranularity.DEFAULT.getBuckets());
+            final PriceGranularity defaultPriceGranularity = PriceGranularity.DEFAULT;
+            return Json.mapper.valueToTree(ExtPriceGranularity.of(defaultPriceGranularity.getPrecision(),
```

Comment:
This line could be extracted to a separate method and reused twice within a _populatePriceGranularity_ method.

Reply:
Going even further a static factory method could be defined in ExtPriceGranularity:
```
public static ExtPriceGranularity from (PriceGranularity priceGranularity){
    return new ExtPriceGranularity(priceGranularity.getPrecision(), priceGranularity.getRanges());
}
```
and used here and in AmpRequestFactory where extension is created using default price granularity.